### ISSUE TITLE
Entrypoint command missing on django app

### ISSUE THREAD

**Issue by general03** [microsoft/vscode-docker#2334]
Hi,

I configured my `launch.json` file in my project

```
{
    "configurations": [
        {
            "name": "Docker: Python - Django",
            "type": "docker",
            "request": "launch",
            "preLaunchTask": "docker-run: debug",
            "python": {
                "pathMappings": [
                    {
                        "localRoot": "${workspaceFolder}/app",
                        "remoteRoot": "/usr/src/app"
                    }
                ],
                "port": 8000,
                "projectType": "django",
                "django": true,
            },
        }
    ]
}

```

With this task.json
```

{
    "version": "2.0.0",
    "tasks": [
        {
            "type": "docker-build",
            "label": "docker-build",
            "platform": "python",
            "dockerBuild": {
                "tag": "myapp:latest",
                "dockerfile": "${workspaceFolder}/app/Dockerfile",
                "context": "${workspaceFolder}",
                "pull": true
            }
        },
        {
            "type": "docker-run",
            "label": "docker-run: debug",
            "dependsOn": [
                "docker-build"
            ],
            "python": {
                "args": [
                    "runserver",
                    "0.0.0.0:8000"
                ],
                "file": "manage.py"
            },
            "dockerRun": {
                "ports": [
                    {
                        "hostPort": 8000,
                        "containerPort": 8000
                    }
                ],
                "volumes": [
                    {
                        "containerPath": "/usr/src/app",
                        "localPath": "${workspaceFolder}/app"
                    }
                ]
            }
        }
    ]
}
```
The build works well without error and my container is up.
But when the build run is executed it miss the `entrypoint` command like I put in the previous file `python manage.py runserver 0.0.0.0:8000`, I see in the console this automatic command

`> docker run -dt -P --name "myapp-dev" --label "com.microsoft.created-by=visual-studio-code" -v "/home/general03/.vscode/extensions/ms-python.python-2020.8.109390/pythonFiles/lib/python/debugpy:/debugpy:ro" --entrypoint "python" "myapp:latest" <`

What I miss ??

Thanks

Docker extension 1.6.0
Python extension v2020.8.109390

Version: 1.49.1
Commit: 58bb7b2331731bf72587010e943852e13e6fd3cf
Date: 2020-09-16T23:23:33.049Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 4.19.0-10-amd64

**Comment by bwateratmsft** [comment]
@general03 This is by design. The `docker-run` tasks are meant for debugging; in order to debug the app at launch time--instead of attaching later--it is necessary to start the container without starting the app inside of it. So, we use `python` as the entrypoint, so that the container stays running but isn't doing anything. Then, when the debug task resolves, it is handed over to the Python extension's debugger, which will actually launch the app inside the container, with the debugger attached from the very beginning. The arguments in the `docker-run` task's `python` section are used as the arguments to launch it.

The debugging flow is covered in https://code.visualstudio.com/docs/containers/debug-python which is also linked in the Python Dockerfile.

**Comment by general03** [comment]
Thanks @bwateratmsft 

But I don't understand why my breakpoint is not reachable, my container started right but nothing append (error or something else). The blue loading run in the runbar without end
I also check the output log in VSCode without error found 

![image](https://user-images.githubusercontent.com/2324694/93893584-16040680-fcee-11ea-8f6d-2670ee314f04.png)

**Comment by bwateratmsft** [comment]
Is there any output in the Terminal window for "Python Debug Console":
![image](https://user-images.githubusercontent.com/36966225/93896481-f3a6c480-fcbe-11ea-96e3-2cddfeda3e72.png)

Or in the Output window from Python:
![image](https://user-images.githubusercontent.com/36966225/93896447-e8ec2f80-fcbe-11ea-929a-48358bc50c7c.png)

Or the debug console:
![image](https://user-images.githubusercontent.com/36966225/93896522-028d7700-fcbf-11ea-94b4-4d68d211da3d.png)

**Comment by general03** [comment]
@bwateratmsft you are a genius ... 
The #2323 help me to launch the debug right

Thanks again

**Comment by bwateratmsft** [comment]
@general03 Did you also need to add `"debugAdapterHost"` to your launch config?

**Comment by general03** [comment]
Yes, because I'm on linux ?

**Comment by bwateratmsft** [comment]
Gotcha. Yeah, we definitely didn't intend for that to be a necessary step. I've fixed the code path in #2331; in the future adding that `"debugAdapterHost"` will not be necessary.
